packages/headless/src/features/facets/generic/facet-actions.ts

```diff
@@ -0,0 +1,3 @@
+import {createAction} from '@reduxjs/toolkit';
+
+export const clearAllFacetValues = createAction('facet/clearAllValues');
```

Comment:
Following on the comments on the conversation page, I think `deselectAllFacets` could be a better name since `Facets` is plural, whereas in `deselectAllFacetValues`, `Facet` is singular.